tagstudio/src/qt/ts_qt.py

```diff
@@ -1321,6 +1364,11 @@ def open_library(self, path):
         self.cur_query: str = ""
         self.selected.clear()
         self.preview_panel.update_widgets()
+        if (
+            self.settings.contains("autosave")
+            and self.settings.value("autosave") == True
```

Comment:
I'm suspecting this wont work, because by default the value you get from settings is string, but you're comparing it to bool